src/devices/Buzzer/Buzzer.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Pwm;
+using System.Device.Pwm.Drivers;
+using System.IO;
+using System.Threading;
+
+namespace Iot.Device.Buzzer
+{
+    /// <summary>
+    /// Simple buzzer binding. Supports buzzers with ground, vcc and signal pins as well as buzzers with only ground and vcc pins.
+    /// </summary>
+    public class Buzzer : IDisposable
+    {
+        private readonly int _buzzerPin;
+        private readonly int _pwmChannel;
+        private readonly PwmController _pwmController;
+
+        /// <summary>
+        /// Create Buzzer class instance with output on specified pin with specified channel.
+        /// </summary>
+        /// <param name="pinNumber">The GPIO pin number in case of a software PWM. The chip in case of a hardware PWM.</param>
+        /// <param name="pwmChannel">The channel to use in case of a hardware PWM.</param>
+        public Buzzer(int pinNumber, int pwmChannel)
```

Comment:
I'd recommend instead of catching exceptions etc just take PwmController and channel as an input.

Reply:
having both ctors would be nice

Reply:
Added new constructor with three parameters next to old one.